DES: Added ValidatorLogic.java

PuravUdayDesai opened the pull request:
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Number: #1553

PuravUdayDesai committed:
Merge pull request #1 from TesseractCoding/master

Sync Master Repository to Forked Repository

PuravUdayDesai committed:
Added DESController.java

PuravUdayDesai committed:
Moved DESController.java under des/

PuravUdayDesai committed:
Added DESCryptographyLogic.java

PuravUdayDesai committed:
Added DESCryptographyModels.java

PuravUdayDesai committed:
Added InitialPermutationLogic.java

PuravUdayDesai committed:
Added InverseInitialPermutationLogic.java

PuravUdayDesai committed:
Added PBoxLogic.java

PuravUdayDesai committed:
Added SBoxLogic.java

PuravUdayDesai committed:
Added ValidationException.java

PuravUdayDesai committed:
Moved ValidationException.java under des/exception/

PuravUdayDesai committed:
Added GeneralMethods.java

PuravUdayDesai committed:
Moved GeneralMethods.java under des/general/

PuravUdayDesai committed:
Added KeyGenerator.java

PuravUdayDesai committed:
Moved KeyGenerator.java under des/key/

PuravUdayDesai committed:
Added KeyGenerationModels.java

PuravUdayDesai committed:
Added ValidatorLogic.java

PuravUdayDesai committed:
Moved ValidatorLogic.java under des/validator/

PuravUdayDesai commented:
@HarshCasper please review the pull request once, if any changes/improvements/modifications needed please let me know about the respective.

Sequence Number: 12

Issue Number: #1553 
Pull Request Number:  #2089

HarshCasper commented:
Please have a look at my review here and make the same changes: https://github.com/TesseractCoding/NeoAlgo/pull/2078

atarax665 commented:
@PuravUdayDesai Are you still working on this ?

PuravUdayDesai commented:
> @PuravUdayDesai Are you still working on this ?

Sorry @atarax665 I just saw the comments and changes requested today, I'll update the following as soon as possible by tomorrow.
Really sorry for keeping you waiting

PuravUdayDesai committed:
Updated DESController.java

Reduced redundant code for printing the plain text, cipher text and decrypted text.

PuravUdayDesai committed:
Updated GeneralMethods.java

Added printTextArray method which is used in main(String[])

atarax665 commented:
Closing this PR due to inactivity.

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #2 from PuravUdayDesai/1553-des-controller

1553 des controller

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #3 from PuravUdayDesai/1553-des-cryptography-logic

1553 des cryptography logic

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #4 from PuravUdayDesai/1553-des-cryptography-models

1553 des cryptography models

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #5 from PuravUdayDesai/1553-des-initial-permutation

1553 des initial permutation

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #6 from PuravUdayDesai/1553-des-inverse-initial-permutation

1553 des inverse initial permutation

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #7 from PuravUdayDesai/1553-des-p-box

1553 des p box

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #8 from PuravUdayDesai/1553-des-s-box

1553 des s box

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #9 from PuravUdayDesai/1553-validation-exception

1553 validation exception

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #10 from PuravUdayDesai/1553-des-general

1553 des general

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai committed:
Merge pull request #12 from PuravUdayDesai/1553-des-key-generator

1553 des key generator

PuravUdayDesai committed:
Removed Redundant Comments and HTML contents of Javadoc

PuravUdayDesai commented:
@atarax665 please reopen the pull request

HarshCasper commented:
Closing this PR because of no updates.
